docs/design.md

```diff
@@ -169,24 +172,29 @@ Many classes in this layer make use of proxy pattern, i.e. they only connect to
 ### Policies
 
 Access control:
+
 + Although this component provides methods that are relevant to access control (e.g. providing user information), the access control check itself does not happen in this component. The UI is expected to check access control (using `GateKeeper` class) before calling a method in the logic component.
 
 API for creating entities:
+
 + Null parameters: Causes an assertion failure.
 + Invalid parameters: Throws `InvalidParametersException`.
 + Entity already exists: Throws `EntityAlreadyExistsException` (escalated from Storage level).
 
 API for retrieving entities:
+
 + Attempting to retrieve objects using `null` parameters: Causes an assertion failure.
 + Entity not found:
-  - Returns `null` if the target entity not found. This way, read operations can be used easily for checking the existence of an entity.
+  + Returns `null` if the target entity not found. This way, read operations can be used easily for checking the existence of an entity.
 
 API for updating entities:
+
 + Update is done using `*UpdateOptions` inside every `*Attributes`. The `UpdateOptions` will specify what is used to identify the entity to update and what will be updated.
 + Entity not found: Throws `EntityDoesNotExistException`.
-+ Invalid parameters: Throws `InvalidParametersException`.
+- Invalid parameters: Throws `InvalidParametersException`.
```

Comment:
Sorry, this change was overlooked. At first when I was putting together this PR, I thought I might update the files for consistency(removing trailing spaces, changing all bullet points to use the same bullet point rather than (+, -, * interchangeably), separating headers from content). For the ease of looking through the PR, I thought I had reverted it to the original, but I missed these bullet points. Will revert.